.changeset/blue-plants-mate.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fuel-ts/account": patch
+---
+
+feat: add `getMessageByNonce` to `Provider`
```

Comment:
I missed this initially, but it should've been a patch because it doesn't break anything.